lib/jsontableschema.rb

```diff
@@ -28,6 +28,7 @@
 require "jsontableschema/types/string"
```

Comment:
Yeah, that's it, this is the main file that requires everything needed by the gem. There are other patterns (i.e. we could modularise, so you could only require `jsonschema/table` or `jsonschema/schema`), but this is the generally accepted pattern, especially for small, focussed gems.